docs/Appendix-J.rst

```diff
@@ -1,102 +1,147 @@
+Overview
+--------
+The POWER-Up software can run on OpenPOWER or x_86 architecture.
 
-Save Container Files
---------------------
+The PowerAI Enterprise Software Install Module provides for rapid installation of the PowerAI Enterprise software to a cluster of Power8 or Power9 servers.
+The install module creates a web based software installation server on one of the cluster nodes or another node with access to the cluster.
+The software server is populated with repositories and files needed for installation of PowerAI Enterprise.
+Once the software server is setup, installation scripts orchestrate the software installation to one or more client nodes.
+The POWER-Up software installer does not currently support installation of PowerAI Enterprise onto the node running the POWER-Up software installer.
+If it is necessary to install PowerAI Enterprise onto the node running the POWER-Up software, this can be done manually or can be accomplished by running the POWER-Up software on an additional node in the cluster.
+Hint: A second POWER-Up server can be quickly prepared by replicating the repositories from the first POWER-Up server.
 
-#. Note container name from LXC status::
+Set up of the POWER-Up Software Installer Node
+----------------------------------------------
 
-    user@origin-host:~$ sudo lxc-ls -f
+POWER-Up Node  Prerequisites;
 
-#. Archive LXC files::
+#. The POWER-Up software installer currently runs under RHEL 7.2 or above.
 
-    user@origin-host:cluster-genesis/scripts $ ./container_save.sh [container_name]
+#. The user account used to run the POWER-Up software needs sudo privileges.
 
-#. Save config.yml, inventory.yml, and known_hosts files::
+#. Enable access to the Extra Packages for Enterprise Linux (EPEL) repository. (https://fedoraproject.org/wiki/EPEL#Quickstart)
 
-    origin-host:<cluster-genesis>/config.yml
-    origin-host:/var/oprc/inventory.yml
-    origin-host:<cluster-genesis>/playbooks/known_hosts
+#. Enable the optional and extras repositories.
 
-Prepare New Host
-----------------
+    On Power8::
 
-#. Install git
+    $ subscription-manager repos --enable rhel-7-optional-rpms --enable rhel-7-extras-rpms
 
-    - Ubuntu::
+    On Power9::
 
-        user@new-host:~$ sudo apt-get install git
+    $ subscription-manager repos --enable=rhel-7-for-power-9-optional-rpms --enable=–enable=rhel-7-for-power-9-extras-rpms
 
-    - RHEL::
+#. Insure that there is at least 40 GB of available disk space in the partition holding the /srv directory::
 
-        user@new-host:~$ sudo yum install git
+    $ df -h /srv
 
-#. From your home directory, clone Cluster Genesis::
+Install the POWER-Up software::
 
-    user@new-host:~$ git clone https://github.com/open-power-ref-design-toolkit/cluster-genesis
+    $ sudo yum install git
 
-#. Install the remaining software packages used by Cluster Genesis and
-   setup the environment::
+    $ git clone https://github.com/open-power-ref-design-toolkit/power-up -b software-install-b1
```

Comment:
They'll also need to run `./scripts/install.sh` and `source ./scripts/setup-env`. We could link to the 'docs/Installing.html' page? (but then they might forget to checkout the 'software-install-b1' branch)

Reply:
Good catch.  For the moment I'd like to keep this as simple to use as possible for the user.  They already have to jump from Knowledge center to here, so I'd like to minimize the jumping around here and put everything they need right in Appendix-J.    (The risk of course is that we make a mistake and leave something out :-) )